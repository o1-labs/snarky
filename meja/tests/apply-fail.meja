let a : _ = fun (f : _ -> _, x : unit) => {
  f (15);
  f (x);
};
