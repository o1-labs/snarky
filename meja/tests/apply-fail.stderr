File "tests/apply-fail.meja", line 1, characters 33-37:
Error: Incompatable types '_' and 'unit':
Cannot unify 'int' and 'unit'.

